src/generate-stories-script.ts

```diff
@@ -33,7 +33,9 @@ async function toImportFn(stories: string[]) {
       logger.warn(`Cannot process ${ext} file with storyStoreV7: ${relativePath}`);
     }
 
-    return `  '${toImportPath(relativePath)}': async () => import('/${relativePath}')`;
+    return `  '${toImportPath(
+      relativePath,
+    )}': async () => import('${process.cwd()}/${relativePath}')`;
```

Comment:
nitpick, unnecessary `async`

Reply:
haha, copied from the storybook sources, but no problem, will remove it, thanks!